.travis.yml

```diff
@@ -9,7 +9,38 @@ cache:
   directories:
     - $HOME/.m2
 
-script: ./mvnw clean verify
+env:
+  matrix:
+    - SPRING_VERSION=4.1.0.RELEASE
```

Comment:
Is it really necessary to build with all these versions, doesn't that slow down testing a lot? I would expect the patch releases to keep the api compatible.